kibana-reports/rendering-engine/headless-chrome/README.md

```diff
@@ -45,8 +45,12 @@ headless-chrome.sh chrome-version-SHA (arch_name (arm64))
       |-libGLESv2.so
       |-libGLESv2.so.TOC
 # Additional libaries:
-  libnss3-dev
-  fonts-liberation
-  libfontconfig
-  (installation command example : sudo apt update && apt install libnss3-dev fonts-liberation libfontconfig)
+- Ubuntu needs additional dependencies to run chromium 
+```
+sudo apt install -y libnss3-dev fonts-liberation libfontconfig1
+```
+- RedHat/CentOS needs additional dependencies to run chromium
```

Comment:
maybe add Amazon Linux 2 here too? (AL2 doesn't actually need font packages. It needs the default font to be selected, which can be fixed by the same font install command for redhat/centos)

Reply:
goot point. will add